api/src/controllers/auth.ts

```diff
@@ -83,7 +112,7 @@ router.post(
 			role: null,
 		};
 
-		const authenticationService = new AuthenticationService({
+		const authenticationService = new (getAuthService(accountability))({
```

Comment:
This doesn't work because `accountability` isn't set from the middleware in this controller.